kubernetes-kit-starter/src/test/java/com/vaadin/kubernetes/starter/sessiontracker/serialization/SpringTransientHandlerVaadinScopesTest.java

```diff
@@ -1,9 +1,10 @@
-package com.vaadin.azure.starter.sessiontracker.serialization;
+package com.vaadin.kubernetes.starter.sessiontracker.serialization;
 
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import org.assertj.core.api.Assertions;
```

Comment:
Not sure what happened, but we still have the static imports from `Assertions` so this shouldn't be necessary.
```suggestion
```